macros/models/_sector/dex/enrich_dex_trades.sql

```diff
@@ -0,0 +1,79 @@
+{% macro enrich_dex_trades(
+    base_trades = null
```

Comment:
should we get rid of `base_` here too and replace to `stg_`?

Reply:
yep, good catch, pushing shortly